qa/tasks/cephfs/test_volumes.py

```diff
@@ -733,6 +741,48 @@ def test_subvolume_resize_infinite_size_future_writes(self):
             raise RuntimeError("expected filling subvolume {0} with {1} file of size {2}MB "
                                "to succeed".format(subvolname, number_of_files, file_size))
 
+    def test_subvolume_info(self):
+        # tests the 'fs subvolume info' command
+
+        subvol_md = ["subvolpath", "bytes_quota", "bytes_pcent", "data_pool", "createdAt"]
+
+        # create subvolume
+        subvolume = self._generate_random_subvolume_name()
+        self._fs_cmd("subvolume", "create", self.volname, subvolume)
+
+        # get subvolume metadata
+        subvol_info = json.loads(self._get_subvolume_info(self.volname, subvolume))
+        if len(subvol_info) == 0:
+            raise RuntimeError("Expected the 'fs subvolume info' command to list metadata of subvolume")
+        else:
```

Comment:
is there a need for an else clause here?

Reply:
Not necessary. Removed